main.py

```diff
@@ -54,6 +54,9 @@ def __init__(self):
         self.fastbootreboot = Gtk.Button(label='Reboot to Fastboot')
         self.fastbootreboot.connect("clicked", self.on_fastboot_click)
         hbox.pack_start(self.fastbootreboot, True, True, 0)
+        self.fastbootdreboot = Gtk.Button(label='Reboot to FastbootD ( Dynamic Devices Only)')
```

Comment:
Please change the string to Reboot to FastbootD only.

Reply:
Sure, Will ammend this change.